types/powermeter.yaml

```diff
@@ -1,5 +1,26 @@
 description: Powermeter types
 types:
+  SignedMeterValue:
+    description: Representation of a signed meter value
+    type: object
+    additionalProperties: false
+    required:
+      - signed_meter_data
+      - signing_method
+      - encoding_method
+    properties:
+      signed_meter_data:
+        description: Base64 encoded signed meter data
```

Comment:
Does this necessarily have to be Base64? I think we can keep things more generic here, since we have `encoding_method` below anyway...